components/breadcrumb/src/index.js

```diff
@@ -45,7 +45,8 @@ const BreadcrumbListItem = styled('li')({
     position: 'absolute',
     top: '-1px',
     bottom: '1px',
-    left: ((CHEVRON_ALTITUDE_CALCULATED * -2) + CHEVRON_SIZE + CHEVRON_BORDER_WIDTH),
+    left:
+      CHEVRON_ALTITUDE_CALCULATED * -2 + CHEVRON_SIZE + CHEVRON_BORDER_WIDTH,
```

Comment:
this change i'm not keen on - the removing of the outer brackets is OK, but the ones around the multiplication i feel are useful to make it clearer what the calculation is

i generally prefer brackets in maths to clarify, as to me it makes the maths easier to follow

(had this code been written without brackets in the first place then i wouldn't expect this to change)

Reply:
don't think prettier has a config option for this https://prettier.io/docs/en/options.html

Reply:
yeah - that sucks

this has been raised as an issue in prettier a few times.  unfortunately it seems that there is no solution for now, and that allowing this may not be possible given how prettier is written